framework/src/utils/MatrixTools.C

```diff
@@ -31,7 +31,7 @@ int inverse(const std::vector<std::vector<Real> > & m, std::vector<std::vector<R
     for (const auto & matrix_entry : rowvec)
       A.push_back(matrix_entry);
 
-  int error = inverse(A, n);
+  int error = inverse(A, (int) n);
```

Comment:
Looking at http://www.netlib.org/lapack/lapack-3.1.1/html/dgetri.f.html I'd say ignore the signedness. `n` is expected to be >= 0 anyways and no output is written to `n`.